modal/_utils/grpc_utils.py

```diff
@@ -198,6 +198,7 @@ async def retry_transient_errors(
     total_timeout: Optional[float] = None,  # timeout for the entire function call
     attempt_timeout_floor=2.0,  # always have at least this much timeout (only for total_timeout)
     retry_warning_message: Optional[RetryWarningMessage] = None,
+    base_metadata: list[tuple[str, str]] = [],
```

Comment:
```suggestion
    metadata: list[tuple[str, str]] = [],
```

really minor nit, but what do you think about calling it `metadata` instead of `base_metadata`? From the caller's perspective I think they might not understand what `base_` means (`base_metadata` seems to suggest that there should also be a `metadata` parameter somewhere).